.gitignore

```diff
@@ -4,3 +4,4 @@ svgomg-code-coverage.html
 npm-debug.log
 test-summary.html
 tests/data-compare/*.svg
+.DS_Store
```

Comment:
Hah you know, this is the first time i have ever noticed they actually add it into the view. I am so used to their being tabs/spaces on the lines i am reviewing so it always looks like it is in a gutter!